Sources/muterCore/TestReporting/TestSuiteOutcome.swift

```diff
@@ -37,7 +37,16 @@ extension TestSuiteOutcome {
     static private func logContainsTestFailure(_ testLog: String) -> Bool {
         let entireTestLog = NSRange(testLog.startIndex... , in: testLog)
         let numberOfFailureMessages = testFailureRegEx.numberOfMatches(in: testLog, options: [], range: entireTestLog)
-        return numberOfFailureMessages > 0 || testLog.contains("** TEST FAILED **")
+        return numberOfFailureMessages > 0 ||
+            testLog.contains(testFailedMessage(from: .xcodebuild)) ||
+            testLog.contains(testFailedMessage(from: .buck))
+    }
+
+    static private func testFailedMessage(from binaryType: BinaryType) -> String {
+        switch binaryType {
+        case .xcodebuild: return "** TEST FAILED **"
+        case .buck: return "TESTS FAILED: "
+        }
```

Comment:
Maybe we could replace it with a simple regular expression, like `TEST(S)? FAILED`